src/dtx/dtx_rpc.c

```diff
@@ -131,7 +131,7 @@ dtx_req_cb(const struct crt_cb_info *cb_info)
 
 	dout = crt_reply_get(req);
 	if (dra->dra_opc == DTX_COMMIT) {
-		dra->dra_committed += dout->do_misc;
+		*dra->dra_committed += dout->do_misc;
```

Comment:
minor nit, I really prefer using parenthesis.  Otherwise it makes my brain hurt figuring out what we are dereferencing.

Reply:
sure, will fix it when refresh the patch.

Reply:
(*dra->dra_committed) += dout->do_misc could be safer.